docs/core/deploying/trimming/incompatibilities.md

```diff
@@ -3,23 +3,29 @@ title: Known trimming incompatibilities
 description: Identify patterns and frameworks that are known to have problems with trimming
 author: agocke
 ms.author: angocke
-ms.date: 10/08/2021
+ms.date: 11/8/2023
 ---
 # Known trimming incompatibilities
 
-There are some patterns that are known to be incompatible with trimming. Some of these patterns might become compatible as tooling improves or as libraries make modifications to become trimming compatible.
+This article lists patterns that are incompatible with trimming with the current tooling.
 
 ## Reflection-based serializers
 
 Alternative: Reflection-free serializers.
 
-Many uses of reflection can be made trimming-compatible, as described in [Introduction to trim warnings](fixing-warnings.md). However, serializers tend to have complex uses of reflection. Many of these uses can't be made analyzable at build time. Unfortunately, the best option is often to rewrite the system to use source generation instead.
+Many uses of reflection can be made trimming-compatible, as described in [Introduction to trim warnings](fixing-warnings.md). However, serializers tend to have complex uses of reflection. Many of these uses can't be made analyzable at build time. Unfortunately, the best option is often to rewrite the system to use source generation.
 
-Popular reflection-based serializers and their recommended alternatives:
+The following table lists popular reflection-based serializers and their recommended alternatives:
 
-- **Newtonsoft.Json**. Recommended alternative: [source generated System.Text.Json](../../../standard/serialization/system-text-json/source-generation.md)
-- **System.Configuration.ConfigurationManager**. Recommended alternative: [source generated Microsoft.Extensions.Configuration](https://github.com/dotnet/runtime/issues/44493)
-- **System.Runtime.Serialization.Formatters.Binary.BinaryFormatter** Recommended alternative: [Migrate away from BinaryFormatter serialization due to its security and reliability flaws.](../../compatibility/serialization/7.0/binaryformatter-apis-produce-errors.md#recommended-action)
+| Serializers | Alternative |
+| :-- | :---------: |
+| **Newtonsoft.Json** | [Source generated `System.Text.Json`](../../../standard/serialization/system-text-json/source-generation.md)    |
+| **System.Configuration.ConfigurationManager** | [Source generated `Microsoft.Extensions.Configuration`](https://github.com/dotnet/runtime/issues/44493) |
```

Comment:
@ericstj @eiriktsarpalis @tarekgh - do we have official docs for the ConfigurationBinder source generator? I'm not sure pointing normal .NET devs to this issues is going to lead them to success in using it.

Reply:
@layomia _do we have official docs for the ConfigurationBinder source generator?_